# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/moz.build
# Commit: 8196045d38b5
# Full Hash: 8196045d38b50e04313725215c36ba1618c50486
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Create an ipdl struct called WebrtcProxyConfig, to cut down on the arg count on lots of functions. r=mjf,mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45562
# ==============================================================================

diff -r 6e1eba5c88ad -r 8196045d38b5 netwerk/ipc/moz.build
--- a/netwerk/ipc/moz.build	Wed Sep 18 14:27:42 2019 +0000
+++ b/netwerk/ipc/moz.build	Tue Sep 17 18:19:04 2019 +0000
@@ -54,6 +54,7 @@
   IPDL_SOURCES += [
       '../../media/mtransport/ipc/PStunAddrsRequest.ipdl',
       '../../media/mtransport/ipc/PWebrtcTCPSocket.ipdl',
+      '../../media/mtransport/ipc/WebrtcProxyConfig.ipdlh',
   ]
   EXPORTS.mozilla.net += [
       '../../media/mtransport/ipc/NrIceStunAddrMessageUtils.h',
